src/cryptsetup/cryptsetup.c

```diff
@@ -24,10 +24,14 @@
 
 /* internal helper */
 #define ANY_LUKS "LUKS"
+/* as in src/cryptsetup.h */
+#define CRYPT_SECTOR_SIZE 512
+#define CRYPT_MAX_SECTOR_SIZE 4096
 
 static const char *arg_type = NULL; /* ANY_LUKS, CRYPT_LUKS1, CRYPT_LUKS2, CRYPT_TCRYPT or CRYPT_PLAIN */
 static char *arg_cipher = NULL;
 static unsigned arg_key_size = 0;
+static unsigned arg_sector_size = CRYPT_SECTOR_SIZE;
```

Comment:
This line should probably be enclosed in "#if/#endif" so as not to get
```
[649/1610] Generating sanitize-address-fuzzers with a custom command.
../src/cryptsetup/cryptsetup.c:34:17: warning: ‘arg_sector_size’ defined but not used [-Wunused-variable]
 static unsigned arg_sector_size = CRYPT_SECTOR_SIZE;
                 ^
```

Reply:
@xnox could you enclose the definition of `arg_sector_size` in `#if/#endif`? The warning is still there.

Reply:
I've created PR #9936.

Reply:
ah, thanks, I missed this comment entirely.